src/L2/L2LockingPosition.sol

```diff
@@ -21,6 +21,7 @@ contract L2LockingPosition is Initializable, Ownable2StepUpgradeable, UUPSUpgrad
     uint256 private nextId;
 
     /// @notice Mapping of locking position ID to LockingPosition entity.
+    // slither-disable-next-line uninitialized-state
```

Comment:
For each disabled warning please add a comment why we (can) disable it.
See [L2Staking](https://github.com/LiskHQ/lisk-contracts/blob/development/src/L2/L2Staking.sol) or [L2LockingPosition](https://github.com/LiskHQ/lisk-contracts/blob/development/src/L2/L2LockingPosition.sol) contracts as an example.